ts/src/bitteam.ts

```diff
@@ -0,0 +1,2395 @@
+         * @see https://bit.team/trade/api/documentation#/CMC/getTradeApiCmcOrderbookPair
+         * @param {string} symbol unified symbol of the market to fetch the order book for
+         * @param {int} [limit] the maximum amount of order book entries to return (default 100, max 200)
+         * @param {object} [params] extra parameters specific to the bitteam api endpoint
+         * @returns {object} A dictionary of [order book structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-book-structure} indexed by market symbols
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'pair': market['id'],
+        };
+        const response = await this.publicGetTradeApiCmcOrderbookPair (this.extend (request, params));
+        //
+        //     {
+        //         "timestamp": 1701166703285,
+        //         "bids": [
+        //             [
+        //                 2019.334988,
+        //                 0.09048525
+        //             ],
+        //             [
+        //                 1999.860002,
+        //                 0.0225
+        //             ],
+        //             ...
+        //         ],
+        //         "asks": [
+        //             [
+        //                 2019.334995,
+        //                 0.00899078
+        //             ],
+        //             [
+        //                 2019.335013,
+        //                 0.09833052
+        //             ],
+        //             ...
+        //         ]
+        //     }
+        //
+        const timestamp = this.safeInteger (response, 'timestamp');
+        const orderbook = this.parseOrderBook (response, symbol, timestamp);
+        return orderbook;
+    }
+
+    async fetchOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Order[]> {
+        /**
+         * @method
+         * @name bitteam#fetchOrders
+         * @description fetches information on multiple orders made by the user
+         * @see https://bit.team/trade/api/documentation#/PRIVATE/getTradeApiCcxtOrdersofuser
+         * @param {string} symbol unified market symbol of the market orders were made in
+         * @param {int} [since] the earliest time in ms to fetch orders for
+         * @param {int} [limit] the maximum number of  orde structures to retrieve (default 10)
+         * @param {object} [params] extra parameters specific to the bitteam api endpoint
+         * @param {string} [params.type] the status of the order - 'active', 'closed', 'cancelled', 'all', 'history' (default 'all')
+         * @returns {Order[]} a list of [order structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
+         */
+        await this.loadMarkets ();
+        let type = this.safeString (params, 'type');
+        if (type === undefined) {
```

Comment:
we can replace these 5 lines with 

```
const type = this.safeString(params, 'type', 'all')
```
We don't need to omit because `type` is the exchange-specific key